corehq/apps/es/es_query.py

```diff
@@ -153,6 +153,16 @@ def __init__(self, index=None, for_export=False):
             }
         }
 
+    def clone(self):
```

Comment:
:+1: love it - I could see us moving away from `deepcopy` at some point, and it's better to have that implementation detail abstracted away